zipkin-storage/cassandra/src/main/resources/zipkin2-schema-indexes.cql

```diff
@@ -33,20 +62,19 @@ CREATE TABLE IF NOT EXISTS zipkin2.span_by_service (
     AND speculative_retry = '95percentile'
     AND comment = 'Secondary table for looking up span names by a service name.';
 
-DROP INDEX IF EXISTS zipkin2.span_annotation_query_idx;
-
-CREATE CUSTOM INDEX IF NOT EXISTS ON zipkin2.span (annotation_query) USING 'org.apache.cassandra.index.sasi.SASIIndex'
-   WITH OPTIONS = {
-    'mode': 'PREFIX',
-    'analyzed': 'true',
-    'analyzer_class':'org.apache.cassandra.index.sasi.analyzer.DelimiterAnalyzer',
-    'delimiter': '░'};
-
-CREATE CUSTOM INDEX IF NOT EXISTS ON zipkin2.span (l_service) USING 'org.apache.cassandra.index.sasi.SASIIndex'
-   WITH OPTIONS = {'mode': 'PREFIX'};
-
-CREATE CUSTOM INDEX IF NOT EXISTS ON zipkin2.trace_by_service_span (duration) USING 'org.apache.cassandra.index.sasi.SASIIndex'
-   WITH OPTIONS = {'mode': 'PREFIX'};
+CREATE TABLE IF NOT EXISTS zipkin2.remote_service_by_service (
```

Comment:
is adding the tables `trace_by_service_remote_service` and `remote_service_by_service` the only actual change in the file's diff?

Reply:
yes, the other changes were moving the indexing near the tables they indexed as it was confusing me (at the cost of confusing you in this review.. sorry)